vignettes/using-rcoins.Rmd

```diff
@@ -0,0 +1,95 @@
+---
+title: "Tracing continuous streets using rcoins"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Tracing continuous streets using rcoins}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  eval = FALSE
+)
+```
+
+```{r setup}
+library(rcoins)
+library(sf)
+library(ggplot2)
```

Comment:
Should `ggplot2` be added to the DESCRIPTION file under Suggests?

Reply:
Yes, added